### PR TITLE
Add sentence to Courses Page doc about search

### PR DESCRIPTION
**Pull request by thallada**
We added a new feature to the Courses Page in Insights: a search input that can filter the course list by course id and name.

You can see the new feature right now on stage: https://stage-insights.edx.org/courses/

FYI @katymyw

### CODE REVIEW

File: docs/en_us/dashboard/source/Courses_Page.rst [hunk 1]
```diff
@@ -34,6 +34,9 @@ Computations`.  Note that the enrollment data that the computations include is
 the same as the summary metrics presented in the :ref:`Enrollment_Activity`
 report.
 
+To find a specific course in the course list, use the search input to filter the
+table to Courses which have Course Names or Course IDs that match the query.
```
Comment (dsjen) [inline docs/en_us/dashboard/source/Courses_Page.rst]: Nit: it looks like "course" isn't capitalized in "course name" and "course ID" a few lines above, https://github.com/edx/edx-analytics-dashboard/pull/624/files#diff-f9273ab8625577952ddda601f630dab8R29.  I'm not sure which would be the standard way.

**Review by dsjen — APPROVED** [review]

File: docs/en_us/dashboard/source/Courses_Page.rst (hunk 1 above)
Comment (srpearce) [reply docs/en_us/dashboard/source/Courses_Page.rst]: I think we can just go with this:

If you want to view a specific course or courses, enter any part of the course name or course ID in the **Find a Course** field and click the search icon.

Comment (thallada) [reply docs/en_us/dashboard/source/Courses_Page.rst]: Sounds good. @srpearce I replaced the sentence with your suggestion.

**Comment by thallada** [comment]
@srpearce everything look good with this now? I think I need one more thumbs up before I merge.

**Review by srpearce — APPROVED** [review]
👍

**Pull request merged**